src/test/java/racinggame/domain/RacingCarTest.java

```diff
@@ -0,0 +1,71 @@
+package racinggame.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class RacingCarTest {
+
+    @Test
+    void 자동차의_이름_글자수_5자_이하_실패() {
+        String carName = "carcar1";
+
+        assertThrows(IllegalArgumentException.class, () -> Validator.validateNameLength(carName));
+    }
+
+    @Test
+    void 자동차의_이름_글자수_5자_이하_성공() {
+        String carName = "car1";
+
+        assertDoesNotThrow(() -> Validator.validateNameLength(carName));
```

Comment:
```suggestion
        assertDoesNotThrow(() -> new RacingCar(carName));
```
이렇게 테스트가 가능하도록 수정하면 어떨까요?

`RacingCar`의 생성자 테스트를 하기위해서 `List<RacingCar>`를  다른 객체의 책임으로 만들 수 있겠네요.

Reply:
감사합니다!! 반영했습니다